README.md

```diff
@@ -49,101 +49,18 @@ The production build of your application, located in the <code>dist</code> d
 ```
 "appInfo": {
     "loki": {
-      "appName": [The Loki app that you plan to deploy to],
```

Comment:
@andrew-waller @andrewhwaller I found the "name" appellations confusing. These are changed to "codeName" to differentiate from "pageName" which is a name a human would recognize.

Reply:
Works for me.